easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.5-Python-3.7.4.eb

```diff
@@ -0,0 +1,41 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Conda'
+
+name = 'MAGeCK-VISPR'
+version = '0.5.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://bitbucket.org/liulab/mageck-vispr/'
+description = """MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
+  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
+  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend."""
+
+source_urls = ['https://bitbucket.org/liulab/%(namelower)s/get']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7e6867c68316ed35f059127cca8f78e6a0a609c274f9448579575449b11095e2']
+
+dependencies = [
+    ('Miniconda3', '4.7.10', '', True),
+    ('Python', '3.7.4', '-GCCcore-8.3.0')
```

Comment:
Having MAGeCK-VISPR on `SYSTEM` and this dependency on Python will cause issues. If there is any other module of Python already loaded, loading MAGeCK-VISPR will fail. The solution might be as easy as removing the dependency on Python though, as it is already installed by conda in any case
```
[...]
    python-3.7.8               |h6f2ec95_1_cpython        53.1 MB  conda-forge
    python-dateutil-2.8.1      |             py_0         220 KB  conda-forge
    python-irodsclient-0.8.2   |             py_0          69 KB  conda-forge
[...]
```